### PR TITLE
Fix OpenAPI specs

### PR DESCRIPTION
**Pull request by lava**
Fix some syntax errors that prevented the correct API display on the website, as identified by the online validator.

### CODE REVIEW

File: web/openapi/openapi.yaml [hunk 1]
```diff
@@ -90,7 +87,7 @@ paths:
           description: Invalid query string or invalid limit.
     post:
       summary: Export data
-      description: Export data from VAST according to a query. The query must be a valid expression in the VAST Language (see https://vast.io/docs/understand/query), followed by an optional pipeline.
+      description: Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/query-language) followed by an optional pipeline string.
```
Comment (mavam) [inline web/openapi/openapi.yaml:90]: This is not correct. Presumably this came from a merge conflict?

File: web/openapi/openapi.yaml [hunk 2]
```diff
@@ -153,7 +150,7 @@ paths:
   /export/with-schemas:
     post:
       summary: Export data with schema information
-      description: "Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/language) The data is returned grouped by schema.\n"
+      description: "Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/query-language) The data is returned grouped by schema.\n"
```
Comment (mavam) [inline web/openapi/openapi.yaml:153]: See above

File: web/openapi/openapi.yaml [hunk 3]
```diff
@@ -19,7 +19,7 @@ paths:
   /export:
     get:
       summary: Export data
-      description: Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/language)
+      description: Export data from VAST according to a query. The query must be a valid expression in the VAST Query Language. (see https://vast.io/docs/understand/query-language)
```
Comment (mavam) [inline web/openapi/openapi.yaml:22]: see below

**Review by dominiklohmann — APPROVED** [review]
It renders well for me locally with these changes.

**Pull request merged**